roles/libvirt_manager/tasks/deploy_layout.yml

```
@@ -175,6 +175,8 @@
       }}
     pub_key: "{{ pub_ssh_key.content | b64decode }}"
     priv_key: "{{ priv_ssh_key.content | b64decode }}"
+    _vm_img: >-
+      {{ vm }}.qcow2
```

```suggestion
    _vm_img: {{ vm }}.qcow2
```
why don't use this simpler form?

---

Copy Paste, out of interested I checked the blame and historically the line was longer[1]

Then it was pushed to two lines to not be so long, then it was shortened without going back to a single line and then I Copy/Pasted it without thinking to change to a single line :P

[1] https://github.com/openstack-k8s-operators/ci-framework/pull/1151